bitnami/jaeger/templates/agent/service.yml

```diff
@@ -0,0 +1,84 @@
+  sessionAffinityConfig: {{- include "common.tplvalues.render" (dict "value" .Values.agent.service.sessionAffinityConfig "context" $) | nindent 4 }}
+  {{- end }}
+  {{- if or (eq .Values.agent.service.type "LoadBalancer") (eq .Values.agent.service.type "NodePort") }}
+  externalTrafficPolicy: {{ .Values.agent.service.externalTrafficPolicy | quote }}
+  {{- end }}
+  ports:
+    - name: compact
+      port: {{ .Values.agent.service.ports.compact }}
+      targetPort: {{ .Values.agent.containerPorts.compact }}
+      {{- if and (or (eq .Values.agent.service.type "NodePort") (eq .Values.agent.service.type "LoadBalancer")) (not (empty .Values.agent.service.nodePorts.compact)) }}
+      nodePort: {{ .Values.agent.service.nodePorts.compact }}
+      {{- else if eq .Values.agent.service.type "ClusterIP" }}
+      nodePort: null
+      {{- end }}
+    - name: binary
+      port: {{ .Values.agent.service.ports.binary }}
+      targetPort: {{ .Values.agent.containerPorts.binary }}
+      {{- if and (or (eq .Values.agent.service.type "NodePort") (eq .Values.agent.service.type "LoadBalancer")) (not (empty .Values.agent.service.nodePorts.binary)) }}
+      nodePort: {{ .Values.agent.service.nodePorts.binary }}
+      {{- else if eq .Values.agent.service.type "ClusterIP" }}
+      nodePort: null
+      {{- end }}
+    - name: config
+      port: {{ .Values.agent.service.ports.config }}
+      targetPort: {{ .Values.agent.containerPorts.config }}
+      {{- if and (or (eq .Values.agent.service.type "NodePort") (eq .Values.agent.service.type "LoadBalancer")) (not (empty .Values.agent.service.nodePorts.config)) }}
+      nodePort: {{ .Values.agent.service.nodePorts.config }}
+      {{- else if eq .Values.agent.service.type "ClusterIP" }}
+      nodePort: null
+      {{- end }}
+    - name: zipkin
+      port: {{ .Values.agent.service.ports.zipkin }}
+      targetPort: {{ .Values.agent.containerPorts.zipkin }}
+      {{- if and (or (eq .Values.agent.service.type "NodePort") (eq .Values.agent.service.type "LoadBalancer")) (not (empty .Values.agent.service.nodePorts.zipkin)) }}
+      nodePort: {{ .Values.agent.service.nodePorts.zipkin }}
+      {{- else if eq .Values.agent.service.type "ClusterIP" }}
+      nodePort: null
+      {{- end }}
+    - name: admin
+      port: {{ .Values.agent.service.ports.admin }}
+      targetPort: {{ .Values.agent.containerPorts.admin }}
+      {{- if and (or (eq .Values.agent.service.type "NodePort") (eq .Values.agent.service.type "LoadBalancer")) (not (empty .Values.agent.service.nodePorts.admin)) }}
+      nodePort: {{ .Values.agent.service.nodePorts.admin }}
+      {{- else if eq .Values.agent.service.type "ClusterIP" }}
+      nodePort: null
+      {{- end }}
+    {{- if .Values.agent.service.extraPorts }}
+    {{- include "common.tplvalues.render" (dict "value" .Values.agent.service.extraPorts "context" $) | nindent 4 }}
+    {{- end }}
+  selector: {{- include "common.labels.matchLabels" . | nindent 4 }}
```

Comment:
Missing selectors here

Reply:
Still not done

Reply:
Done in [cba6d52](https://github.com/bitnami/charts/pull/13480/commits/cba6d52f4905953abd32b6a8b6fafdc9f6b4cf8d)